common/lib/dependabot/pull_request_creator/github.rb

```diff
@@ -439,8 +439,9 @@ def add_assignees_to_pull_request(pull_request)
           pull_request.number,
           assignees
         )
-      rescue Octokit::NotFound
-        # This can happen if a passed assignee login is now an org account
+      rescue Octokit::UnprocessableEntity, Octokit::NotFound
+        # Octokit::UnprocessableEntity - This can happen if an invalid assignee was passed
```

Comment:
🔎 What do you think about narrowing in on the error? That could be done by checking the contents of message before bailing, and re-raising if it's not what we intend to swallow.

Reply:
Unfortunately I haven't been able to reproduce this issue locally, so I don't have a fixture file for it. I came up with 2e60d50076bf5a1d97a1403deabeb7d00ad5d2a3 based on the sentry event 
![Screenshot 2024-05-31 at 12 45 24 PM](https://github.com/dependabot/dependabot-core/assets/12107187/636bad6b-0ff3-407b-a4bf-c9904c5843d6)